gatorgrade/main.py

```diff
@@ -0,0 +1,18 @@
+import typer
+
+app = typer.Typer(add_completion=False)
+file = "gatorgrade.yml"
+
+
+@app.callback(invoke_without_command=True)
+def gatorgrade(file: str):
+    """Runs the GatorGrader checks that are defined in the gatorgrade.yml file."""
+
+
+@app.command()
+def generate(force: bool = typer.Option(False, "--force", "-F")):
```

Comment:
Generally it's a good idea to have all options be lower-case -- upper-case options are normally used only when there is already a lower-case version that would conflict. This makes typing the command faster, since the user doesn't need to type a capital letter. There are a fair amount of exceptions in various command line tools, however.

Reply:
Thats true, I made it uppercase as it is what the generate team suggested to me, though if the short version is for ease of access it makes more sense for it to be lowercase. I changed it to be lowercase and I'll follow up with the generate team about the change!